app/controllers/crimes_controller.rb

```diff
@@ -1,9 +1,13 @@
 class CrimesController < ApplicationController
   # Jackson
   def index
-    @crimes = Crime.all
+
     get_crime_types
     get_crime_areas
+
+    @query = params[:keywords].split(" ") if params[:keywords]
```

Comment:
could have: maybe include a regex that also accounts for commas:
" "
", "
","